geometry_msgs/Point targetPosition
---
bool isAllowed